composables/use-log-event.ts

```diff
@@ -1,8 +1,11 @@
+import { setUser } from '@sentry/nuxt'
```

Comment:
Prevent possible name collision 
```suggestion
import { setUser as setSentryUser } from '@sentry/nuxt'
```